doc/source/getting_started/install.rst

```diff
@@ -193,25 +193,25 @@ Visualization
 
 Installable with ``pip install "pandas[plot, output-formatting]"``.
 
-========================= ================== ================== =============================================================
```

Comment:
Could you undo all the changes in this file? They don't appear related to the linked issue

Reply:
Some commits in this PR [#60457](https://github.com/pandas-dev/pandas/issues/60457)  were mistakenly added but are meant for issue https://github.com/pandas-dev/pandas/pull/60446.  I am new to open-source contributions and apologize for the confusion. 

Could you please guide me on how I can avoid this mistake in the future?